src/main/java/com/github/zauworks/chatdictionary/command/DictionarySettingCommand.java

```diff
@@ -0,0 +1,218 @@
+package com.github.zauworks.chatdictionary.command;
+
+import com.github.zauworks.chatdictionary.ChatDictionary;
+import com.github.zauworks.chatdictionary.dictionary.DictionaryWord;
+import com.github.zauworks.chatdictionary.util.DataSave;
+import com.github.zauworks.chatdictionary.util.Messages;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DictionarySettingCommand implements CommandExecutor {
+
+    private final String permission = "dictionary.command.setting";
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if (!(sender instanceof Player)) {
+            sender.sendMessage(new Messages().notice("[ChatDictionary Notice]: from the Command Console?"));
+        } else if (!sender.hasPermission(permission)) {
+            sender.sendMessage(new Messages().error("[ChatDirectory Warn]: You do not have the permission."));
+            return false;
+        }
+
+        if (args.length < 2) {
+            sender.sendMessage(new Messages().error("[Error]:Arguments not enough."));
+            return false;
+        }
+
+        String type = args[0];
+        if (type.equalsIgnoreCase("add")) {
+            if (args.length < 3) {
+                sender.sendMessage(new Messages().error("[Error]:Usage: /dictionary add [word] [mean]"));
+                return false;
+            }
+        } else if (type.equalsIgnoreCase("options")) {
+            if (args.length < 4) {
+                sender.sendMessage(new Messages().error("[Error]:Usage: /dictionary options [word] <bold|underline|color|mean> [value]"));
+                return false;
+            }
+        } else if (type.equalsIgnoreCase("config")) {
+            if (args.length < 3) {
+                sender.sendMessage(new Messages().error("[Error]:Usage: /dictionary config reload [word]"));
+                return false;
+            }
+        }
+
+        if (type.equalsIgnoreCase("add") && args.length >= 3) {
```

Comment:
ここの `args` の判定は消しちゃいましょう。
```suggestion
        if (type.equalsIgnoreCase("add")) {
```